_docs/_latest/use/cli/index.md

```diff
@@ -7,30 +7,40 @@ hide:
 
 # {{ page.title }}
 
-Learn about using the CLI and third-party tool queries to work with Forseti Security.
+Learn about using the command-line interface (CLI) and 
+third-party tool queries to work with Forseti Security 
+and its modules.
 
 ---
-Use the command line interface (CLI) to interface with Forseti and its modules. 
 
-* **[Configuring the CLI]({% link _docs/latest/use/cli/cli.md %})**: Learn how to
-  configure the CLI.
+**[Configuring the CLI]({% link _docs/latest/use/cli/cli.md %})**
 
-* **[Use the Inventory]({% link _docs/latest/use/cli/inventory.md %})**: Find out how to use third party
-  tools to query raw Forseti Inventory data.
+Learn how to configure the CLI.
 
-* **[Use models]({% link _docs/latest/use/cli/model.md %})**: Build and use Forseti data models.
+**[Use Inventory]({% link _docs/latest/use/cli/inventory.md %})**
 
-* **[Use the Scanner]({% link _docs/latest/use/cli/scanner.md %})**: Use the Forseti
-  Scanner to audit your environment.
+Use third party tools to query raw Forseti Inventory data.
 
-* **[Use the Notifier]({% link _docs/latest/use/cli/notifier.md %})**: Send notifications
-  using Notifier to send notifications.
+**[Use models]({% link _docs/latest/use/cli/model.md %})**
 
-* **[Use Explain]({% link _docs/latest/use/cli/explain.md %})**: Understand your
-  environment using Explain.
+Build and use Forseti data models.
 
-* **[Use the Forseti Server]({% link _docs/latest/use/cli/server.md %})**: Update the
-  configuration of the Server.
+**[Use Scanner]({% link _docs/latest/use/cli/scanner.md %})**
 
-* **[Use Enforcer]({% link _docs/latest/use/cli/enforcer.md %})**: Correct policy discrepancies
-  using the Enforcer.
\ No newline at end of file
+Audit your environment using Forseti Scanner.
+
+**[Use Notifier]({% link _docs/latest/use/cli/notifier.md %})**
+
+Send notifications using Notifier.
+
+**[Use Explain]({% link _docs/latest/use/cli/explain.md %})**
+
+Understand your environment using Explain.
+
+**[Use Forseti Server]({% link _docs/latest/use/cli/server.md %})**
+
```

Comment:
Can you confirm that the order of these entries in the index.md match the order on the left sidebar?

Reply:
Confirmed, resolved.